### DISCUSSION TITLE
What version of webkit ships with macOS builds?

### DISCUSSION THREAD

**Discussion by keyle** [wailsapp/wails#1470] [Q&A] [answered]
Hi, what version of webkit ships with macOS builds?

I've noticed 'newer' javascript functions such as `replaceAll` didn't appear to work. Or I'm doing something wrong?

If so, that would place it between webkit version 608-610, which would put it back to Q2 2019.

Any plans for upgrading it?

**Comment by stffabi** [comment, marked as answer]
Wails does not ship WebKit within the binaries. It uses the WKWebView (basically a Wrapper around WebKit) component of Apple which is included with macOS itself.
So the WebKit version used depends on the version of macOS on wich the application is running. It might also contain additional fixes or changes that are not open sourced by Apple.
